CCDB/src/runConditionsClient.cxx

```diff
@@ -1,80 +1,84 @@
 /********************************************************************************
  *    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
  *                                                                              *
- *              This software is distributed under the terms of the             * 
- *         GNU Lesser General Public Licence version 3 (LGPL) version 3,        *  
+ *              This software is distributed under the terms of the             *
+ *         GNU Lesser General Public Licence version 3 (LGPL) version 3,        *
  *                  copied verbatim in the file "LICENSE"                       *
  ********************************************************************************/
 /**
  * runConditionsClient.cxx
  *
  * @since 2013-04-23
- * @author D. Klein, A. Rybalchenko
+ * @author D. Klein, A. Rybalchenko, R. Grosso, C. Kouzinopoulos
  */
 
 #include <iostream>
 
+#include "CCDB/ConditionsMQClient.h"
 #include "FairMQLogger.h"
 #include "FairMQParser.h"
 #include "FairMQProgOptions.h"
 
-#include "CCDB/ConditionsMQClient.h"
+#ifdef NANOMSG
+#include "FairMQTransportFactoryNN.h"
+#else
+#include "FairMQTransportFactoryZMQ.h"
+#endif
 
```

Comment:
If you use the new approach (as in FairMQ examples & flp2epn examples), you don't need these lines, the transport is chosen via a cmd option `--transport` (default it zeromq), which is handled internally.

But yeah, besides the flp2epn examples, there are likely other outdated devices in the repository.